api/Element.json

```diff
@@ -5874,7 +5894,7 @@
               }
             ],
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
IE and Edge both had support for an ms-prefixed and/or a webkit-prefixed version of the fullscreen API, but never shipped an unprefixed version. Can you revert the fullscreen-related changes and deal with them separately? It will be pretty messy and take a lot of iteration to get just right.

I will add a comment that says just "revert" to each line, so that it's easy to see if anything was missed, not because I'm very rude :)